### PR TITLE
Helm3 updats 

### PR DESCRIPTION
**Pull request by costinm**
The helm3 test makefile is broken, also added installing the ingress. 

Probably should be moved to samples.

### CODE REVIEW

**Review by dgn — COMMENTED** [review]
Should we have an integration test that attempts to install using helm3, to make sure this doesn't break again?

**Comment by costinm** [comment]
Yes, we should have a test - but I'm not familiar enough with the infrastructure and I'm focusing on a different set of issues - I'm fixing this just so it's not broken..

**Comment by costinm** [comment]
Reviewers: what do you think of moving this to samples (similar with the cert creation sample - also driven by a makefile ) ?

**Comment by costinm** [comment]
BTW - I don't remember who volunteered (or was volunteered) to lead helm3 integration. More volunteers welcomed !

**Review by howardjohn — APPROVED** [review]

**Comment by howardjohn** [comment]
testing + moving to samples both sound like good ideas

**Comment by nrjpoddar** [comment]
@costin we would put more time and effort into it. Please work with @jacob-delgado on better testing or support for Helmv3

File: tests/istio.mk [hunk 1]
```diff
@@ -1,32 +1,61 @@
...
-		-f manifests/charts/charts/global.yaml \
-		 --set meshConfig.defaultConfig.proxyMetadata.DNS_CAPTURE="" \
-		 --set meshConfig.defaultConfig.proxyMetadata.DNS_AGENT=""
+	helm3 template --include-crds istio-base manifests/charts/base | kubectl apply -f -
 
+helm3/master:
+	helm3 upgrade -i -n istio-system istio-16 manifests/charts/istio-control/istio-discovery \
+		--set global.tag=${TAG} --set global.hub=${HUB} \
+        --set revision=master \
+		-f manifests/charts/global.yaml \
```
Comment (nrjpoddar) [inline tests/istio.mk:13]: What’s your plan for globals.yaml long term?

**Review by nrjpoddar — COMMENTED** [review]
What’s in globals.yaml that cannot be moved to one of the charts?

File: tests/istio.mk (hunk 1 above)
Comment (costinm) [reply tests/istio.mk:13]: Get rid of them, of course. We're very close - John added 'mesh config' to values.yaml, and we have only 2 charts - istiod and gateways. 

But it's not my top priority- if someone has time, just remove the flag and see what breaks/changes - and move whatever is found to mesh config or values.yaml.

**Comment by costinm** [comment]
@howardjohn - any test suite where I could call the makefile target to install helm3 canary on top of a istioctl-install ? It's probably the most important test, so far most problems I've seen are when something else is already installed. 

Should the 'fresh install' go to a new test suite or can it be added to an existing one ? 

I would need some help from people familiar with the suites. I can also add this to the 
stability test - having a helm3 installed rev may be good.

**Pull request merged**